tests/test-run.sh

```diff
@@ -80,6 +80,13 @@ assert_file_has_content runtime-fpi "^runtime=runtime/org\.test\.Platform/$ARCH/
 
 ok "run a runtime"
 
+run_sh org.test.Platform env | grep service_host_ >runtime-os-release
+while read -r line; do
+assert_file_has_content runtime-os-release ${line}
```

Comment:
Surely `"service_host_${line}"`?

Reply:
Unfortunately, an assertion this simple will not work if you get the quoting/unquoting right.

Reply:
Updated, using eval to remove quoting